### PR TITLE
magnus changes for BYE text

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5866,11 +5866,26 @@ a=msid:-
...
+          current RTP stream to "m=" line table. For this type of
+          pattern it is common that the RTCP packets and information
+          is repeated, either periodically (e.g. SDES items), or for a
+          duration (e.g. BYE), thus suppression of repetitions can be
+          considered. Examples of these are:<list style="hanging">
+
+		  <t hangText="Goodbye (BYE) RTCP Packet:">This <xref
+		  target="RFC3550">RTP/RTCP protocol</xref> mechanism
+		  indicates that a particular RTP stream is leaving the RTP
+		  session. Thus, a most relevant event to inform the handler
```
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: Irrelevant side note but ... In the context of SDP offer / answer, it's pretty much baffling why this is "a most relevant" event. "irrelevant" I could buy

Comment (aboba) [inline draft-ietf-rtcweb-jsep.xml]: @fluffy  Have to agree that the original BYE text is easier to understand, though it is WRONG in an important respect, which is that a BYE (or a timeout) needs to result in removal of a "latched" entry in the ssrc table.  Magnus's replacement text doesn't fix the problem though.

**Comment by ekr** [comment]
+1 to Fluffy, Bernard.

**Comment by juberti** [comment]
Agree with the above - will try to rework and fix the problem indicated by @aboba

**Comment by gloinul** [comment]
So, most of the text is discussing the "design pattern" the RTCP Bye falls under. Then the BYE event's "relevant" text may not be good writing. However, I fail to see why BYE becomes irrelevant in a O/A context. Yes, if one terminates the session, it doesn't matter. But for updates that changes the configuration and removes an RTP stream that might not be explicitly signalled in the SDP, then saying BYE matters.

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Plan will be to simply address case of removing from SSRC table when receiving a BYE.

Comment (aboba) [inline draft-ietf-rtcweb-jsep.xml]: Removal from the "latching table" also needs to occur in event of a timeout.  Note that signalled SSRCs cannot be removed due to a BYE or timeout.

**Comment by juberti** [comment]
@aboba, @fluffy, PTAL. If we need to handle something other than BYE, e.g. timeouts, let's handle that separately.

**Comment by juberti** [comment]
@aboba, any further comments before I land this?

**Pull request merged**